### ISSUE TITLE
W2: Finalize logit experimentation

### ISSUE THREAD

**Issue by cemrtkn** [cemrtkn/conversational-collapse#2]
# Overview

Following observations at #1, this week will attempt to broaden insight into logits. 

## Tasks
- [ ] Run experiment with more conversation starters than 3 (which was the amount used in the first week). Look for insight.
- [ ] Experiment with varying generation arguments such as: temperature, top_p, max_new_tokens?
- [ ] Investigate the relation between the similarity metrics and entropy
- [ ] Investigate relationship with per token perplexity and entropy. Luis thinks that per token perp can tell where the entropy is coming from.
